code/datums/emotes.dm

```diff
@@ -55,7 +55,7 @@
 		if(!M.client || isnewplayer(M))
 			continue
 		var/T = get_turf(user)
-		if(M.stat == DEAD && M.client && (M.client.prefs.toggles & CHAT_GHOSTSIGHT) && !(M in viewers(T, null)))
+		if(M.stat == DEAD && isobserver(M) && M.client && (M.client.prefs.toggles & CHAT_GHOSTSIGHT) && !(M in viewers(T, null)))
```

Comment:
why is `M.stat == DEAD` necessary?

Reply:
In most parts of the code the two are called in unisson ; it is probably redundant.